src/attr/_make.py

```diff
@@ -373,11 +380,19 @@ def wrap(cls):
                 # It might not actually be in there, e.g. if using 'these'.
                 cls_dict.pop(ca_name, None)
             cls_dict.pop("__dict__", None)
+            old_cls = cls
 
             qualname = getattr(cls, "__qualname__", None)
             cls = type(cls)(cls.__name__, cls.__bases__, cls_dict)
             if qualname is not None:
                 cls.__qualname__ = qualname
+            for item in cls.__dict__.values():
+                closure_cells = getattr(item, '__closure__', [])
+                if not closure_cells:
```

Comment:
Please use a specific one – any reason you use [] and not None in 390 anyways?  I guess you could remove the whole if statement?